docs/DEVELOPMENT.md

```diff
@@ -35,6 +34,32 @@ npx tauri dev
 
 Automatic recompiling on save is enabled for both the Rust and the Typescript/Vue code.
 
+If you want to build, do all of the above (except tauri dev) and:
+
+```sh
+cd ../src-tauri
+```
+
+```sh
+npm run tauri build
+```
+
+This will build the executable and bundles, such as AppImage or .deb.
+
+To build just the executable, edit [tauri.conf.json](https://github.com/R2NorthstarTools/FlightCore/blob/c102fdcf8ef00e3c5c46fb9ded37e09b7e92e483/src-tauri/tauri.conf.json#L18) in the same folder.
+
+Change active from `true` to `false`, and bundles won't be included afterward.
+
+To disable the updater change [this value](https://github.com/R2NorthstarTools/FlightCore/blob/c102fdcf8ef00e3c5c46fb9ded37e09b7e92e483/src-tauri/tauri.conf.json#L54) to false.
+
+All of this can be simplified into a one-line command:
+
+```sh
+cd src-tauri; sed -i '18s/.*/      "active": false,/' tauri.conf.json; sed -i '54s/.*/      "active": false,/' tauri.conf.json; cd ..; npm install && cd src-vue; npm install && cd ../src-tauri; npm run tauri build
```

Comment:
I see that you're using raw line numbers here, does it mean that this documentation becomes invalid if we update `tauri.conf.json`?

Reply:
Yes, and the same for links mentioning it. I'll see if I can change it to find it.

Reply:
Yes, and the same for links mentioning it. I'll see if I can change it to find.

Reply:
To be fully independent of line numbers I'd just use `jq` to parse/edit the JSON instead. Although I don't really see the need to put the script in here in the first place to be honest ^^

Reply:
Ok, I'll delete.